sharktank/sharktank/examples/export_paged_llm_v1.py

```diff
@@ -128,15 +148,17 @@ def _(
 
     generate_batch_prefill(4)
     generate_batch_decode(4)
+    config = generate_params_json(hp, [4], [4])
     print("GENERATED!")
 
     for name, ep in fxb.programs.items():
         print(f"EXPORT {name}:\n{ep}")
 
     print("Exporting")
     output = export(fxb)
-    print(f"Saving to '{args.output}'")
-    output.save_mlir(args.output)
+    print(f"Saving to '{args.output_mlir}'")
+    output.save_mlir(args.output_config)
+    json.dump(config, open("/tmp/batch_llama_v1.json", "w"))
```

Comment:
Use flag here
```suggestion
    json.dump(config, open(args.output_config, "w"))
```

(and maybe we should ensure that the directory exists for both of these flags, creating it as needed)